lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -70,7 +70,13 @@ def routing_spec?(args)
         end
 
         def glob_for((described_class, method_name))
-          "#{expected_path(described_class)}#{name_glob(method_name)}*_spec.rb"
+          glob = []
+          unless spec_suffix_only?
+            glob << expected_path(described_class)
+            glob << name_glob(method_name)
+          end
+          glob << '*_spec.rb'
+          glob.join
```

Comment:
This looks pretty unreadable to me.
Do you mind to split it into two methods? E.g.
```ruby
def glob_for((described_class, method_name))
  return glob_for_spec_suffix_only?(described_class, method_name) if spec_suffix_only?

  "#{expected_path(described_class)}#{name_glob(method_name)}*_spec.rb"
end

def glob_for_spec_suffix_only?(described_class, method_name)
  "*_spec.rb"
end
```

Reply:
I don't mind at all. I've updated and re-pushed this branch up.